packages/clerk-js/src/ui/UserProfile/VerifyWithLink.tsx

```diff
@@ -20,13 +20,20 @@ export const VerifyWithLink = (props: VerifyWithLinkProps) => {
   const card = useCardState();
   const profileContext = useUserProfileContext();
   const { startMagicLinkFlow } = useMagicLink(email);
-
-  React.useEffect(() => {
-    startVerification();
-  }, []);
```

Comment:
:question: Did we remove the `useEffect` hook? How is startVerification triggered?

Reply:
Of course not! What was I thinking :p

Reply:
Of course not! What was I thinking? Fixed.